### PR TITLE
just fixing a typo i noticed after signing up

### PR DESCRIPTION
**Pull request by chasd00**
## What kind of change does this PR introduce?

Bug fix

## What is the current behavior?

On the home page after signing up under Explore our other products the Realtime card says "Listen to your to PostgreSQL.."

## What is the new behavior?

Realtime card now says "Listen to your PostgreSQL.."

## Additional context

Fixed the text in two files but i have a feeling it only really needs to be fixed in one (i'm not a font-end dev)

### CODE REVIEW

**Comment by chasd00** [comment]
just realized my own fix is wrong but you guys get the idea :)

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 1]
```diff
@@ -212,7 +212,7 @@ const NewProjectPanel: FC<Props> = ({}) => {
                   </div>
                   <div className="flex md:min-h-[50px] xl:min-h-[75px]">
                     <p className="text-sm text-scale-1000">
-                      Listen to your to PostgreSQL database in realtime via websockets.
+                      Listen your to PostgreSQL database in realtime via websockets.
```
Comment (gregnr) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:215]:
```suggestion
                      Listen to your PostgreSQL database in realtime via websockets.
```

File: apps/docs/data/repos/realtime.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "site_admin": false
   },
   "html_url": "https://github.com/supabase/realtime",
-  "description": "Listen to your to PostgreSQL database in realtime via websockets. Built with Elixir.",
+  "description": "Listen your to PostgreSQL database in realtime via websockets. Built with Elixir.",
```
Comment (gregnr) [inline apps/docs/data/repos/realtime.json:28]:
```suggestion
  "description": "Listen to your PostgreSQL database in realtime via websockets. Built with Elixir.",
```

**Review by gregnr — APPROVED** [review]
Nice catch, thanks @chasd00 😃

**Pull request merged**